pkgs/development/python-modules/git-filter-repo/default.nix

```diff
@@ -13,10 +15,21 @@ buildPythonPackage rec {
 
   disabled = pythonOlder "3.5";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-/hdT4Y8L1tPJtXhoyAEa59BWpuurcGcGOWoV71MScl4=";
-  };
+  srcs = [
+    (fetchPypi {
+      inherit pname version;
+      hash = "sha256-/hdT4Y8L1tPJtXhoyAEa59BWpuurcGcGOWoV71MScl4=";
+    })
+    (fetchFromGitHub {
+      name = "docs";
+      owner = "newren";
+      repo = pname;
+      rev = "01ead411966a83dfcfb35f9d2e8a9f7f215eaa65";
+      hash = "sha256-Z/3w3Rguo8sfuc/OQ25eFbMfiOHjxQqPY6S32zuvoY4=";
+    })
```

Comment:
I think it would be easier to understand declaring a variable named `docs` rather than using srcs.

Reply:
Can we add `passthru.updateScripts`?
Automatic package updates would cause inconsistencies between the application and the documentation.
Perhaps you can pass an option like `--version=branch=docs` to nix-update-script.

Reply:
I found that `nix-update` only updates versions specified as `src` attribute. It's not possible to upgrade `docs` attribute, which is our case.
I'll take a look at other projects and check how they upgrade their sources when they have multiple sources.